lib/btrfs_test.pm

```diff
@@ -40,7 +41,34 @@ way to get DBus-less environment is to enter rescue.target via systemctl.
 =cut
 sub snapper_nodbus_setup {
     my ($self) = @_;
-    if (script_run('! systemctl is-active dbus')) {
+    # serial console prints text style as ascii chars in PS1 by default.
+    my $escseq = '[\]\[\(\)\e\d\w\s]*';
+
+    my $rescue_service_config_path = "/usr/lib/systemd/system/rescue.service";
+
+    return unless script_run('! systemctl is-active dbus');
+    if (serial_terminal::select_virtio_console()) {
+
+        # Replace StandardInput in rescue.service file and add TTYPath.
+        script_run "cp $rescue_service_config_path $rescue_service_config_path.sav";
```

Comment:
I think a more common way is to provide a [replacement unit file](https://wiki.archlinux.org/index.php/systemd#Replacement_unit_files) by creating the file /etc/systemd/system/rescue.service or create a drop-in file. Can you use `assert_script_run` by default? We don't want to continue the test when even this copy command fails, right? By default it is suggested to use `assert_script_run` unless you want to explicitly check the return value, e.g. as you do in line 49

Reply:
Thank you for your hint! I wasn't aware of this option. I will try to implement it the mentioned way.